cpp/dolfin/fem/GenericDofMap.cpp

```diff
@@ -217,13 +216,14 @@ void GenericDofMap::ufc_tabulate_dofs(
       const int num_entity_dofs = entity_dofs[d][i].size();
```

Comment:
Maybe use a const reference for `entity_dofs[d][i]` here, as it comes up quite a few times below.